project/yax.scala

```diff
@@ -0,0 +1,93 @@
+import sbt._
+import sbt.Keys._
+import java.io._
+import scala.io.Source
+
+object yax {
+
+  private def process(file: File, lines: List[String], flags: Set[String]): List[String] = {
```

Comment:
I forget the details, tpolecat would be the person to corner. Either about hacks or yaks. Definitely he was slightly horrified at having to write this kind of stuff to support both scalaz and cats.